auto-generated/cluster/stone-prd-rh01/tenants/sfo-security-tenant/appstudio.redhat.com_v1beta1_integrationtestscenario_sfo-enterprise-contract.yaml

```diff
@@ -18,5 +18,5 @@ spec:
     - name: revision
       value: main
     - name: pathInRepo
-      value: pipelines/enterprise-contract.yaml
+      value: pipelines/enterprise-contract-redhat-no-hermetic.yaml
```

Comment:
I think this doesn't really make a difference. The POLICY_CONFIGURATION param above is the bit that matters.

I recommend trying `rhtap-releng-tenant/app-interface-standard`.